src/components/proxmox/GuestSection.tsx

```
@@ -22,7 +22,7 @@ export function GuestSection({ label, guests, expanded, onToggle }: GuestSection
       {/* Section header row */}
       <div
         onClick={onToggle}
-        className={`flex items-center gap-2 pl-10 pr-4 py-2 cursor-pointer ${BORDER} bg-[var(--joy-palette-background-level1)]`}
+        className={`flex items-center gap-2 pl-10 pr-4 py-2 cursor-pointer ${BORDER} bg-[var(--mui-palette-background-level1)]`}
       >
```

Biome (2.4.4)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)